packages/@lwc/template-compiler/src/codegen/static-element-serializer.ts

```diff
@@ -117,21 +133,25 @@ function serializeTextNode(text: Text, useRawContent: boolean): string {
     return templateStringEscape(content);
 }
 
-export function serializeStaticElement(element: Element, preserveComments: boolean): string {
+export function serializeStaticElement(
+    element: Element,
+    preserveComments: boolean,
+    codeGen: CodeGen
+): string {
```

Comment:
If we're passing in the whole `codeGen` then there's no need to pass in `preserveComments` as well. You can get it from the `codeGen`.